docs/releases/2.8.0.rst

```diff
@@ -29,82 +31,162 @@ Major Changes
+    once optionally providing a comment for the suggesters
+
 - Configuration system
 - Plugin framework
 - Comment system
-- amaGama Translation Memory is now queried using CORS
-- Improvements on timeline
+- Added ``pootle_format`` app
+
+  - Added support to have several formats in the same project
+
 - Browse pages:
 
-  - Stats refresh notice has been changed to be less intrusive and more helpful
+  - Refactored stats backend:
+
+    - Removed stats refresh notice
+    - Faster stats retrieval
+
   - Hide disabled items and allow showing them to admin users
   - Show leaderboard on top panel and expanded stats panel
   - Search widget is disabled if user cannot translate
-  - Improvements to the statistics table for overly long filenames and smaller screens
+  - Improvements to the statistics table for overly long filenames and smaller
+    screens
+  - Files dropdown no longer keeps references to empty directories
+  - Fixed issue where "Back" button not always worked
+  - Fixed issue with project dropdown when there are projects without name
+
+- Added team page:
+
+  - Only for languages so far, and only available to language managers
+  - Replaces permissions with roles
+  - Provides direct access to suggestion bulk management
+
+- Removed for performance reasons:
+
+  - Removed statistics from user profiles. Will be brought back in the future.
+  - Removed export view
+  - Removed unperformant "More stats" in admin dashboard
+
+- Removed reports feature
+- Own Pootle localization changes
+
+  - Updated translations. You can still `contribute translation updates for
+    your language <http://pootle.locamotion.org/projects/pootle/>`_.
+  - Now ``compilejsi18n`` is used to compile JavaScript translations into
+    assets, thus requiring ``django-statici18n`` app.
+  - Password reset email is now localizable in Pootle
 
-- Cross-language translation is now restricted to admins since translators
-  don't require them.
 - Upload:
 
-  - Disabled upload for non-PO projects as conflict handling currently only works in PO
+  - Disabled upload for non-PO projects as conflict handling currently only
+    works in PO
   - Admins can upload translations as other user
 
 - New Machine Translation providers:
 
-  - `Caighdeán <https://github.com/kscanne/caighdean/blob/master/API.md>`_ - Irish
+  - `Caighdeán (Irish) <https://github.com/kscanne/caighdean/blob/master/API.md>`_
   - `Welsh <http://techiaith.cymru/api/translation/?lang=en>`_
 
+- Added placeholders check for Plurr-formatted strings
 - Improved RQ usage and new management commands
 - Documentation updates
 
 
 Pootle FS
 ---------
 
-Pootle FS allows to synchronize Pootle against a filesystem, or VCS, allowing
-to handle conflicts and several other situations that previously required
-ad-hoc hacks.
+:ref:`Pootle FS <pootle_fs>` allows to synchronize Pootle against a filesystem,
+or VCS, allowing to handle conflicts and several other situations that
```

Comment:
. Handling conflict resolution and other situations of two files being out of sync